babel.config.js

```diff
@@ -32,6 +32,6 @@ const metro = {
 module.exports = ({caller}) => {
     // For `react-native` (iOS/Android) caller will be "metro"
     // For `webpack` (Web) caller will be "@babel-loader"
-    const runningIn = caller(({name}) => name);
+    const runningIn = caller((args = {}) => args.name);
```

Comment:
NAB but why was this necessary?

Reply:
When storybook loads the babel config there is no `args` object so some things were quietly breaking. Specifically, parsing the docs out of `propTypes` would not work. Maybe I should add some comments so nobody breaks this.

Reply:
Added some comment here about the significance to SB